core/internal/cltest/cltest.go

```diff
@@ -1331,51 +1331,25 @@ func BatchElemMustMatchParams(t *testing.T, req rpc.BatchElem, hash common.Hash,
 	}
 }
 
-type SimulateIncomingHeadsArgs struct {
-	StartBlock, EndBlock int64
-	HeadTrackables       []httypes.HeadTrackable
-	Blocks               *Blocks
-}
-
 // SimulateIncomingHeads spawns a goroutine which sends a stream of heads and closes the returned channel when finished.
-func SimulateIncomingHeads(t *testing.T, args SimulateIncomingHeadsArgs) (done chan struct{}) {
-	t.Helper()
-	lggr := logger.TestLogger(t).Named("SimulateIncomingHeads")
-	lggr.Infof("Simulating incoming heads from %v to %v...", args.StartBlock, args.EndBlock)
-
-	if args.EndBlock > args.StartBlock {
-		if l := 1 + args.EndBlock - args.StartBlock; l > int64(len(args.Blocks.Heads)) {
-			t.Fatalf("invalid configuration: too few blocks %d for range length %d", len(args.Blocks.Heads), l)
-		}
-	}
-
+func SimulateIncomingHeads(t *testing.T, heads []*evmtypes.Head, headTrackables ...httypes.HeadTrackable) (done chan struct{}) {
 	// Build the full chain of heads
-	heads := args.Blocks.Heads
 	ctx, cancel := context.WithTimeout(context.Background(), testutils.WaitTimeout(t))
 	t.Cleanup(cancel)
 	done = make(chan struct{})
 	go func(t *testing.T) {
 		defer close(done)
-		ticker := time.NewTicker(250 * time.Millisecond)
+		ticker := time.NewTicker(200 * time.Millisecond)
```

Comment:
any reason for this change? should this value be derived instead of hard coded?

Reply:
I started fiddling with it when it seemed like the tests were slow or somehow dependent on this, but I don't think it actually matters.

Reply:
Reverted